docs/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+---
+name: PR template
+title: 'PR name in Conventional Commits format (Issue #<Number>)'
+assignees: @me
+---
+
+**Description:**
+
+<SHORT_DESCRIPTION>
+
+Issues:
+
+- Issue #<TICKET_ID>
+
+**Related:**
+
+List of related pull requests:
+
+- none
+
+**Checklist:**
+
+- [ ] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- [ ] the pull request name starts with `fix:`, `feat:`, `feature:`, `chore:`, `hotfix:` or `e2e:`
```

Comment:
It should be mentioned that if PR contains breaking changes it should be mentioned in PR name